action.yaml

```diff
@@ -26,7 +26,24 @@ runs:
     - name: Check out repository code
       uses: actions/checkout@v3
 
+    - name: Get changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v35
+
+    - name: Detect changed dependencies
+      shell: bash
+      run: |
+        passed=false        
+        for file in ${{ steps.changed-files.outputs.all_changed_files }}; do          
+          if [[ $file == *"package.json"* ]] || [[ $file == *"Gemfile"* ]] || [[ $file == *"requirements.txt"* ]]; then
```

Comment:
makes an assumption on the names of deps file. what if a user has reqs.txt as the dep file. we need to filter based on DEPENDENCY_FILES input.